packages/client/components/GitLabScopingSearchResults.tsx

```diff
@@ -42,27 +31,6 @@ const GitLabScopingSearchResults = (props: Props) => {
         $queryString: String!
         $selectedProjectsIds: [ID!]
       ) {
-        ...GitLabScopingSearchResults_query
-      }
-    `,
-    queryRef,
-    {UNSTABLE_renderPolicy: 'full'}
-  )
-
-  const paginationRes = usePaginationFragment<
-    GitLabScopingSearchResultsPaginationQuery,
-    GitLabScopingSearchResults_query$key
-  >(
-    graphql`
-      fragment GitLabScopingSearchResults_query on Query
-        @argumentDefinitions(
-          projectsFirst: {type: "Int", defaultValue: 75}
-          issuesFirst: {type: "Int", defaultValue: 25}
-          projectsAfter: {type: "String"}
-          issuesAfter: {type: "String"}
-          projectIds: {type: "[ID!]", defaultValue: null}
-        )
-        @refetchable(queryName: "GitLabScopingSearchResultsPaginationQuery") {
```

Comment:
This is no longer a refetchable query as we're now just getting a large number of issues and letting the user filter them with the search and menu. This will be refactored in this issue: https://github.com/ParabolInc/parabol/issues/6325